### PR TITLE
Update dependencies and resolve issues and deprecations

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: app/styles/app.scss [hunk 1]
```diff
@@ -239,3 +239,5 @@ body {
 .tt-suggestion:not(:hover) .inactive {
   background-color: rgb(230, 230, 230);
 }
+
+@import "ember-power-select";
```
Comment (czosel) [inline app/styles/app.scss]: why is this import not at the top?

File: package.json [hunk 2]
```diff
@@ -10,8 +10,7 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server --proxy http://localhost:8000",
-    "test": "ember test",
-    "docs": "ember ember-cli-yuidoc"
```
Comment (czosel) [inline package.json]: No more yuidoc? (Has this ever been used?)

Comment (anehx) [reply package.json]: No it hasn't - that's why..

File: app/styles/app.scss (hunk 1 above)
Comment (anehx) [reply app/styles/app.scss]: This is duplicated.. The import is in `app/styles/ember-power-select-custom.scss`.

Comment (anehx) [reply app/styles/app.scss]: Fixed!

**Comment by anehx** [comment]
Hooray!

**Comment by czosel** [comment]
LGTM! :100:

**Pull request merged**